src/lib/get-product-data.ts

```diff
@@ -0,0 +1,23 @@
+import fs from 'fs'
+import path from 'path'
+import moize from 'moize'
+import { LearnProductData, LearnProductSlug } from 'types/products'
+
+function getProductData(product: LearnProductSlug): LearnProductData {
```

Comment:
This type should be `ProductSlug` and `ProductData` so it can be used anywhere.

Reply:
Agreed! Is this a note-to-self to resolve in this PR or a fast-follow?